pom.xml

```diff
@@ -5,7 +5,7 @@
 
 	<groupId>org.matmaul.freeboxos</groupId>
 	<artifactId>freeboxos-client</artifactId>
-	<version>0.3.9</version>
+	<version>0.3.10</version>
```

Comment:
I forgot that I can now directly edit in github :) I am editing and merging, thanks.